apps/omg_child_chain/lib/omg_child_chain/block_queue/core.ex

```diff
@@ -478,6 +478,10 @@ defmodule OMG.ChildChain.BlockQueue.Core do
         _ = Logger.info("Submitted #{inspect(submission)} at: #{inspect(txhash)}")
         :ok
 
+        {:error, %{"code" => -32_000, "message" => "insufficient funds for gas * price + value"}}
```

Comment:
hm, I think this should crash loudly, why silence it out to a warning?

Reply:
Indeed. Forgot to close the pr